pkg/server/utils.go

```diff
@@ -72,34 +66,41 @@ func startSpdkMockupServerOnUnixSocket(rpc *spdkJSONRPC) net.Listener {
 	return ln
 }
 
-func spdkMockServerOnUnixSocket(rpc *spdkJSONRPC, l net.Listener, toSend []string) {
+func spdkMockServerCommunicate(rpc *spdkJSONRPC, l net.Listener, toSend []string) {
 	for _, spdk := range toSend {
+		// wait for client to connect (accept stage)
```

Comment:
Usually, a comment in the code it is an indication that a function is missing. Would it make sense to create a couple of small functions?
```
getConnection(....)
readFromClient(...)
substituteIdInResponse(...)
writeToClient(...)
closeWrite(...)
```

Reply:
opened #264